package.json

```diff
@@ -145,7 +150,7 @@
     "coveralls": "^3.0.2",
     "cross-env": "^5.2.0",
     "dts-bundle-generator": "^1.6.1",
-    "full-icu": "^1.2.1",
+    "full-icu": "^1.3.4",
     "html-loader": "^0.5.5",
     "husky": "^0.14.3",
     "intl": "^1.2.5",
```

Comment:
Since this PR is going to most likely end up with an major bump anyways, how about we take the opportunity to change the `tslint` and `typescript` version as well?

Reply:
Other than the jest-related dependencies every other dependencies are updated.